services/manifest-repo-export-service/pkg/repository/repository.go

```diff
@@ -566,9 +566,17 @@ func (r *repository) ApplyTransformers(ctx context.Context, transaction *sql.Tx,
 	}
 
 	// TODO this will be handled in Ref SRX-PA568W
+	authorEmail, ok := (*transformers[0].GetMetadata())["authorEmail"]
```

Comment:
This function does currently allow multiple transformers, however this is actually never used with more than one.
Please change the function to accept exactly one transformer.